source/administration/config-settings.rst

```diff
@@ -391,14 +370,44 @@ Restrict the permission level required to delete private groups. Deleted groups
 | This feature's ``config.json`` setting is ``"RestrictPrivateChannelDeletion": "all"`` with options ``all``, ``team_admin`` and ``system_admin`` for above settings respectively.   |
 +------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Allow which users to delete messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Restrict the permission level required to delete messages.
+
+**Message authors can delete their own messages, and Administrators can delete any message**: Allow authors to delete their own messages, and allow Administrators to delete any message.
```

Comment:
>Administrators > Admins

Administrators comes from this screenshot: https://pre-release.mattermost.com/api/v3/public/files/4ycrbf6e5fn38bp5yj7him113r/get?h=YTHjE15PkS3zYa78AC4rJhfiTEQEpiAYxFjZEQZN9UY

![image](https://cloud.githubusercontent.com/assets/20325173/23074280/376254ce-f4ed-11e6-91d1-75230697ab8d.png)